global.json

```
@@ -1,13 +1,13 @@
 {
   "tools": {
-    "dotnet": "3.1.200",
+    "dotnet": "5.0.100-preview.8.20417.9",
     "vs": {
       "version": "16.6"
```

Not sure if this will fix the current build failure. If not, ignore the suggestion.


```suggestion
      "version": "16.8"
```